resources/kcp/OWNERS

```diff
@@ -1,10 +1,10 @@
 approvers:
   - PK85
-  - wozniakjan
   - piotrmiskiewicz
   - szwedm
   - ralikio
-  - tillknuesting
+  - jaroslaw-pieszka
+  - ukff
```

Comment:
I think we still miss one person from gophers team.

Reply:
I'm sorry, forgot to add you. Update incoming.